pkg/oc/cli/admin/migrate/storage/storage.go

```diff
@@ -362,7 +362,7 @@ func (t *tokenLimiter) getDuration(n int) time.Duration {
 	reservation := t.rateLimiter.ReserveN(now, n)
 	if !reservation.OK() {
 		// this should never happen but we do not want to hang a worker forever
-		glog.Errorf("unable to get rate limited reservation, burst=%d n=%d limiter=%#v", t.burst, n, *t.rateLimiter)
+		glog.Errorf("unable to get rate limited reservation, burst=%d n=%d", t.burst, n)
```

Comment:
`call of glog.Errorf copies lock value: github.com/openshift/origin/vendor/golang.org/x/time/rate.Limiter contains sync.Mutex`